repose-aggregator/tests/release-verification/src/vagrant/deb/Vagrantfile

```diff
@@ -0,0 +1,172 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+repose_version = ENV['REPOSE_VERSION'] || 'local'
+
+$update_yum = <<SCRIPT
+echo "-------------------------------------------------------------------------------------------------------------------"
+echo "Updating Apt packages"
+echo "-------------------------------------------------------------------------------------------------------------------"
+
+# update package lists from the server
+apt-get update -qq
+#apt-get upgrade -yq
```

Comment:
Should this be uncommented, removed, or commented on the side as to why you might want to manually uncomment it for some condition?